subprojects/core-api/src/main/java/org/gradle/api/artifacts/Configuration.java

```diff
@@ -640,6 +640,24 @@ public String determineName(Configuration c) {
      */
     boolean isCanBeResolved();
 
+    /**
+     * Configures if a configuration can have dependencies declared upon it.
+     *
+     * @since 8.2
+     */
+    @Incubating
+    void setCanBeDeclaredAgainst(boolean allowed);
```

Comment:
This should use the new naming scheme

Reply:
Yes.  I'll update this after the other PR merges.